File: Week_4/problem_4-Sorting.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/219774/problem/H

/* 
Note:
* Don't use built-in-functions.
* try to solve it with bubble sort algorithm or Selection Sort. 
*/


#include <bits/stdc++.h>
using namespace std;

int main()
{
    int n;
    cin >> n;
    int ar[n];

    for (int i = 0; i < n; i++)
    {
        cin >> ar[i];
    }

    int temp;
    for (int i = 0; i < n - 1; i++)
    {
        for (int j = i + 1; j < n; j++)
        {
            if (ar[i] > ar[j])
            {
                temp = ar[i];
                ar[i] = ar[j];
                ar[j] = temp;
            }
        }
    }

    for (int i = 0; i < n; i++)
    {
        cout << ar[i] << " ";
    }

    return 0;
}

File: Week_4/problem_4-Sorting.py
# https://codeforces.com/group/MWSDmqGsZm/contest/219774/problem/H

""" 
Note:
* Don't use built-in-functions.
* try to solve it with bubble sort algorithm or Selection Sort.  
"""


n = int(input())
ar = list(map(int, input().split()))

# Bubble sort algorithm
for i in range(n - 1):
    for j in range(i + 1, n):
        if ar[i] > ar[j]:
            # Swap elements
            temp = ar[i]
            ar[i] = ar[j]
            ar[j] = temp

# Print the sorted array
for num in ar:
    print(num, end=" ")
